openassessment/workflow/api.py

```diff
@@ -182,8 +182,10 @@ def get_workflow_for_submission(submission_uuid, assessment_requirements):
     return update_from_assessments(submission_uuid, assessment_requirements)
 
 
-def update_from_assessments(submission_uuid, assessment_requirements):
-    """Update our workflow status based on the status of peer and self assessments.
+def update_from_assessments(submission_uuid, assessment_requirements, force_update_score=False):
```

Comment:
@explorerleslie - I just want to clarify the product expectations here. Is it true that anytime a staff score is present, that will always be used as the overriding score?